canonical-websites/design.ubuntu.com
# Remove examples section

Removed the examples section from the site.

## QA

 - `./run`
 - Check that "examples" is no longer in the main nav and that all references to the section are no longer present.
